Mlem/Views/Tabs/Search/Results/CommunityResultView.swift

```diff
@@ -50,6 +52,14 @@ struct CommunityResultView: View {
     }
     
     var body: some View {
+        if hasSwipeActions {
+            content
+                .addSwipeyActions(trailing: [subscribeSwipeAction])
+        } else {
+            content
+        }
+    }
```

Comment:
See comment above--it would be nice for this view to not come bundled with swipe actions